### PR TITLE
Fixes #14922: Missing timing logs in ldap queries (especially for dynamic groups) 

### PR DESCRIPTION
**Pull request by ncharles**
https://issues.rudder.io/issues/14922

### CODE REVIEW

File: webapp/sources/rudder/rudder-web/src/main/resources/logback.xml [hunk 1]
```diff
@@ -360,7 +363,7 @@ along with Rudder.  If not, see <http://www.gnu.org/licenses/>.
     ================
     This logger is in charge of all scheduled jobs and batches
   -->
-  <logger name="scheduledJob" level="info" additivity="false">
+  <logger name="scheduledJob" level="debug" additivity="false">
```
Comment (fanf) [inline webapp/sources/rudder/rudder-web/src/main/resources/logback.xml]: should be let to info

**Comment by ncharles** [comment]
Commit modified

**Comment by ncharles** [comment]
OK, merging this PR

**Comment by ncharles** [comment]
OK, merging this PR

**Pull request merged**